agenta-web/src/services/human-evaluations/index.ts

```diff
@@ -0,0 +1,244 @@
+import axios from "@/lib//helpers/axiosConfig"
+import {
+    EvaluationResponseType,
+    Evaluation,
+    GenericObject,
+    CreateCustomEvaluation,
+    ExecuteCustomEvalCode,
+    AICritiqueCreate,
+} from "@/lib/Types"
+import {
+    fromEvaluationResponseToEvaluation,
+    fromEvaluationScenarioResponseToEvaluationScenario,
+} from "@/lib/transformers"
+import {EvaluationFlow, EvaluationType} from "@/lib/enums"
+import {getAgentaApiUrl} from "@/lib/helpers/utils"
+
+export const loadEvaluations = async (appId: string, ignoreAxiosError: boolean = false) => {
```

Comment:
For consistency, let's please use the same naming convention everywhere:

<img width="346" alt="image" src="https://github.com/Agenta-AI/agenta/assets/17180132/e0d56896-9513-45fc-9b8a-2a28400ab084">